proto/pulumi/language.proto

```diff
@@ -444,6 +444,12 @@ message RunPluginRequest{
 
     // The [plugin program](pulumirpc.ProgramInfo) to use.
     ProgramInfo info = 5;
+
+    // The plugin prefix for display purposes.
+    string prefix = 6; 
```

Comment:
Feel like there's a better name than prefix to use here. This will always be a string like "my-provider (resource)" or "cool-policies (analyzer)". That's both name and kind, but "prefix" doesn't really tell you that.

It also might be worthwhile telling RunPlugin what kind is being started, at least for python and node I think we might need that for backcompat analyzer boot up. At which point you could then also pass "name" and then rebuild "prefix" language-host side if needed.

Reply:
100% agree